### ISSUE TITLE
[EN][Glyph][Warlock] Glyph of Unstable Affliction

### ISSUE THREAD

**Issue by Fearyourself** [Atlantiss/BugTracker#9976]
[//]: # (***************************************************************)
[//]: # (** DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED **)
[//]: # (***************************************************************)

[//]: # (On the Bugtracker all titles of reports must be written in English so that everyone, no matter which language they use everyday, could search and check if the problem already has been reported by another player. When describing the problem you can use English as well as Polish and in nearest future - others. It is important to determine at the beginning of a title in which language it is written by using symbols [EN] or [PL].)

[//]: # (Enclose links to things related to the bug using http://wowhead.com or http://cata.openwow.com)
[//]: # (Write your tickets according to the format:)
[//]: # ([EN][Quest][Zone] A Vision of the Past)
[//]: # ([EN][NPC] Thoralius the Wise)
[//]: # ([EN][Spell][Class] Frostfire Bolt)
[//]: # ([PL][Talent][Class] Brain Freeze)
[//]: # ([EN][Glyph][Class] Glyph of Frostfire Bolt)
[//]: # ([EN][Npc][Drop] Thoralius the Wise)
[//]: # ([EN][Web] Armory doesnt work)

**Description**: Back in earlier wow (late tbc iirc) the glyph would only reduce casting time by 0.2s resulting in no damage increase because of the global cooldown (GCD). They changed it so that the glyph would also reduce GCD by 0.2 sec.

**Current behaviour**: It reduces the cast time of UA by 0.2s but not the global cooldown (GCD) resulting in 0.0 damage increase. You don't need fancy addons or anything to test it, it's easily recognizeable.

**Expected behaviour**: It should reduce the cast time and global cooldown of UA by 0.2s

**Steps to reproduce the problem**:

1. No glyphs/gear/buffs (be naked mate)
2. Cast UA (1.5s cast time, gcd 1.5s)
3. Try with glyph
4. Cast UA (1.3 sec cast time, gcd 1.5s)

**Revision**: 5722

Proof:

https://us.battle.net/forums/en/wow/topic/1965618214
Vaydra on 2011/01/17
"(...) Hell back in wrath UA was glitched where it reduced the cast time of UA but not the GCD causing it to be a 0.0 dps increase. I typically would avoid it like the plague =P"

Proof 2:

http://www.wowhead.com/item=42472/charred-glyph#comments
Windle on 2008/10/17 (Patch 3.0.2)
"As of build 9061, this glyph also reduces the GCD caused by UA by 0.2 sec. Thus, casting is not clipped by the GCD, and you can chain a cast with the shortened UA. (...)"

**Comment by formul8r** [comment]
This is absolutely NOT true. Completely false information, it absolute does not reduce the gcd. Please research more before you post these things.
https://us.battle.net/forums/en/wow/topic/16646034943

**Comment by firemagegodx** [comment]
"It reduces the cast time of UA by 0.2s but not the global cooldown (GCD) resulting in 0.0 damage increase."
Well, you're telling us that:
1. Your UA is always on 1s cast time (???)
2. Reducing the cast time of UA should allow you to use Shadow Bolt more.. no? This is a DPS increase, for sure.
Probably one of the more odd things I've read here.

**Comment by Fearyourself** [comment]
The link you provided is from 2015.

I didn't say that my cast time on UA is always 1 sec.
Why are you talking about shadow bolt? I didn't even mention shadow bolt. This bug has absolutely nothing to do with shadow bolt????

**Comment by Fearyourself** [comment]
Are you guys both trolling???

**Comment by firemagegodx** [comment]
Well you did claim something that was false. Same with the link from 2011, where the person in question claims a reduced casting time doesn't increase your damage, which is absurd. Lesser cast time, the faster you can cast other spells, no?
And I mentioned 1s cast because thats the absolutely fastest you can cast something, without having the GCD stopping you from doing more. So the only way it'd be a DPS loss is if the cast would be less than 1s, so you have to wait for your GCD to end after the cast is done. And no, the damage increase would be more significant than 0.000002%. But nevertheless, don't downvote something that is undeniably true, when you don't have all the facts or don't understand.
